tests/spanner_dbapi/test_connection.py

```diff
@@ -11,7 +11,9 @@
 
 # import google.cloud.spanner_dbapi.exceptions as dbapi_exceptions
 
+from google.api_core.exceptions import Aborted
 from google.cloud.spanner_dbapi import Connection, InterfaceError
+from google.cloud.spanner_dbapi.checksum import ResultsChecksum
 from google.cloud.spanner_dbapi.connection import AUTOCOMMIT_MODE_WARNING
 from google.cloud.spanner_v1.database import Database
 from google.cloud.spanner_v1.instance import Instance
```

Comment:
Why these tests are still here? They were copied into `unit` directory, so I suppose they should be erased from this directory?!

Reply:
Good question. Looks like this (and `test_connect`) weren't moved in #532?

https://github.com/q-logic/python-spanner-django/blob/41abaebb6f2e0b1cf16704aa1e394acc5a47e68b/tests/spanner_dbapi/test_connection.py

The test files weren't exactly copied, #532 changed them and added some new tests. E.g. the [version on master now](https://github.com/googleapis/python-spanner-django/blob/927e17846398a111e3cd5bfe878d5497eded0609/tests/unit/spanner_dbapi/test_connection.py) doesn't include `test_transaction_autocommit_warnings`.

@mf2199 can you confirm that you meant to change/remove these tests before removing `tests/spanner_dbapi` in this PR?